install/dc-detect-version.sh

```diff
@@ -10,6 +10,23 @@ echo "${_group}Initializing Docker Compose ..."
 
 # To support users that are symlinking to docker-compose
 dc_base="$(docker compose version &>/dev/null && echo 'docker compose' || echo 'docker-compose')"
+dc_base_standalone="$(docker-compose version &>/dev/null && echo 'docker-compose' || echo '')"
+
+COMPOSE_VERSION=$($dc_base version --short || echo '')
+STANDALONE_COMPOSE_VERSION=$($dc_base_standalone version --short &>/dev/null || echo '')
+
+if [[ -z "$COMPOSE_VERSION" && -z "$STANDALONE_COMPOSE_VERSION" ]]; then
```

Comment:
@BYK `COMPOSE_VERSION` would have been empty if neither `docker compose` nor `docker-compose` existed and checking it would suffice.

But this would work as well.

Reply:
Interesting. How? I mean if we have `docker-compose` somewhere but not `docker compose`, is that not possible?

Reply:
Try this for example:
```
$ dc_base="$(docker NONSENSE version &>/dev/null && echo 'docker compose' || echo 'docker-compose')"
$ COMPOSE_VERSION=$($dc_base version --short || echo '')
$ echo $COMPOSE_VERSION 
2.33.1
```

Reply:
Ah, right missed that part. Can simplify, thanks for pointing it out!